src/hashring.c

```diff
@@ -13,13 +13,15 @@
 struct hashring {
 	list_t backends;
 	void *alloc_data;
+	bool is_monitor_ring;
```

Comment:
Maybe make this a ring_type_t enum or something

typedef enum {
  RING_DEFAULT = 0,
  RING_MONITOR = 1,
} hashring_type_t;